### DISCUSSION TITLE
Announcement: BoTorch will stop publishing a conda package to the pytorch channel

### DISCUSSION THREAD

**Discussion by Balandat** [meta-pytorch/botorch#2613] [General]
**tl;dr: BoTorch will stop publishing a botorch conda package to the `-c pytorch` conda channel. However, there will still be a conda package available on the `-c conda-forge` channel for those users who need it**

Supporting publishing our own conda package has caused an outsized maintenance burden over the years, especially more recently. conda builds are slow, the setup is brittle and often breaks. There were a share of bugs with the conda dependency solver in the past. Recently, our conda tests broke and we haven't been able to figure out why. At this point we can't really justify this added maintenance burden given the limited benefit of publishing the conda package, especially since the `-c conda-forge` channel also publishes a [botorch package](https://anaconda.org/conda-forge/botorch). Finally, [PyTorch is deprecating its official anaconda channel](https://github.com/pytorch/pytorch/issues/138506) for similar reasons. 

BoTorch version 0.12 will have been the last version published to the `-c pytorch` channel. This will reduce maintenance burden and speed up GitHub Actions CI by requiring fewer testing workflows (and thus also save compute capacity).
